scripts/setup/get-icons.js

```diff
@@ -0,0 +1,175 @@
+'use strict';
+const { exec, execSync } = require('child_process');
+const fs = require('fs');
+const chalk = require('chalk');
+const path = require('path');
+
+const fileExists = fs.existsSync('./.resources.json');
+if (!fileExists) {
+  console.log('File ./.resources.json not exists');
+  return;
+}
+
+const resourcesData = JSON.parse(fs.readFileSync('./.resources.json'));
+const iconFolder = 'public/assets/icons/';
+
+if (!resourcesData.hasOwnProperty('repoUrl') && !resourcesData.hasOwnProperty('path')) {
+  return;
+}
+
+class IconManagerBase {
+  constructor(resourcesData, iconFolder, tempFolder) {
+    this.resourcesData = resourcesData;
+    this.iconFolder = iconFolder;
+    this.tempFolder = tempFolder;
+    this.rootPath = '../../';
+  }
+}
+
+class ISparseCheckoutGitV225 extends IconManagerBase {
```

Comment:
Although this is not typescript, but let's still try to follow the style guide https://basarat.gitbook.io/typescript/styleguide#interface

Reply:
Got it! Thanks for the feedback!